File: Nível 1/2377 - Pedágio/main.lua
infos1 = io.read()
infos2 = io.read()

comprimento, pedagio = infos1:match("(%S+)%s+(%S+)")
custoKm, custoPedagio = infos2:match("(%S+)%s+(%S+)")

comprimento, pedagio, custoKm, custoPedagio = tonumber(comprimento), tonumber(pedagio), tonumber(custoKm), tonumber(custoPedagio)

calc = math.floor(comprimento / pedagio)
calc = calc * custoPedagio + (comprimento * custoKm)

print(calc)

File: Nível 1/2377 - Pedágio/main.py
infos1 = input("A: ").split(" ")
infos2 = input("B: ").split(" ")

comprimento, pedagio = int(infos1[0]), int(infos1[1])
custoKm, custoPedagio = int(infos2[0]), int(infos2[1])

calc = comprimento // pedagio
calc = calc * custoPedagio + (comprimento * custoKm)

print(calc)